components/resources/library/src/desktopMain/kotlin/org/jetbrains/compose/resources/ResourceReader.desktop.kt

```diff
@@ -1,15 +1,22 @@
 package org.jetbrains.compose.resources
 
-import java.io.EOFException
-import java.io.IOException
 import java.io.InputStream
 
-internal actual fun getPlatformResourceReader(): ResourceReader = object : ResourceReader {
-    override suspend fun read(path: String): ByteArray {
-        val resource = getResourceAsStream(path)
-        return resource.use { input -> input.readBytes() }
+actual fun getDefaultResourceReader(): ResourceReader =
+    JvmResourceReader.DEFAULT
+
+class JvmResourceReader(
+    private val classLoader: ClassLoader = JvmResourceReader::class.java.classLoader
+) : ResourceReader {
+
+    companion object {
+        val DEFAULT = JvmResourceReader()
```

Comment:
```suggestion
        val Default = JvmResourceReader()
```
the Compose style guidelines are different from Kotlin guidelines

(cc @terrakok, we should take this into account for all future API, even if we already used another style)